sbnana/CAFAna/Core/ISyst.h

```diff
@@ -50,6 +50,15 @@ namespace ana
                        caf::SRSliceProxy* sr,
                        double& weight) const = 0;
 
+    /// \brief Perform the systematic shift
+    ///
+    /// \param sigma   Number of sigma to shift record by
+    /// \param nu      SRTrueInteraction
+    /// \param weight  Scale this weight for reweighting systematics
+    virtual void Shift(double sigma,
+                       caf::SRTrueInteractionProxy* nu,
+                       double& weight) const = 0;
```

Comment:
Just thinking, is it better to have this a pure virtual or as a do-nothing that can be optionally given an override? Anyway, if others are happy with or even prefer this, then fine!